docs/userGuide/syntax/lists.md

```diff
@@ -16,6 +16,131 @@
+
+**The icon's appearance can be further customized by adding a `i-class` attribute.**
+
+<div id="main-example-markbind">
+
+<include src="codeAndOutput.md" boilerplate >
+<variable name="highlightStyle">markdown</variable>
+<variable name="code">
+* Item 1 <br>
+  Item 1 line 2{ icon="/images/deer.jpg" i-width="60px" height="17px" i-class="rounded" }
+* Item 2
+  Item 2 Continue
+  * Item 2.1 { icon="fas-question-circle" i-class="badge rounded-pill my-1 bg-success text-white" }
+  * Item 2.2 { i-class="badge rounded-pill my-1 bg-danger text-white" }
+  * Item 2.3 
+    * Item 2.3.1 { icon="fas-question-circle" i-class="badge rounded my-1 bg-danger text-white" }
+* Item 3
+  * Item 3.1 
+  * Item 3.2 { i-class="badge rounded-pill my-1 bg-primary text-white"}
+  * Item 3.3 { icon="fas-compass" i-class="badge rounded-pill my-1 bg-warning text-white }
+  * Item 3.4
+* Item 4
+  * Item 4.1
+    * Item 4.1.1
+  * Item 4.2 
+</variable>
+</include>
+
+1. Once a `i-class` attribute is assigned to an icon, it will continue apply to icons at the same level, until it is overridden by a different `i-class` attribute (e.g., from Item 2.1 to Item 2.2, and so on to Item 3.2).
+
+</div>
+
+
+**You can apply Markdown's heading and paragraph syntax within the list.**
+
+<include src="codeAndOutput.md" boilerplate >
+<variable name="highlightStyle">markdown</variable>
+<variable name="code">
+
+* #### Heading 1: Overview {icon="/images/overview-d.png" i-width="65px" i-class="rounded" }
+  
+   Content 1: This section provides a summary of the document or topic. 
+   It sets the context and purpose of the content to follow.
+
+   * ##### Heading 1.1: Highlights { icon="/images/highlights-d.png" i-width="65px" i-class="rounded" }
+  
+      Content 1.1: Here we summarize the key points, findings, or features of the main topic. 
+      This could include important data, outcomes, or insights.
+
+* #### Heading 2: Detailed Description { icon="/images/detailed-d.png" i-width="65px" i-class="rounded" }
+    
+   Content 2: This section delves deeper into the topic, offering comprehensive information and detailed explanations.
+   It might also include evidence, examples, or justifications.
+
+* #### Heading 3: Conclusion { icon="/images/conclusion-d.png" i-width="65px" i-class="rounded" }
+  
+    Content 3: The conclusion draws together the main threads of the topic, 
+    summarizing the central points and their implications.
+
+   * ##### Heading 3.1: Final Thoughts
+  
+     Content 3.1: This is a place for final reflections, recommendations, or a look towards future developments or trends.
+
+</variable>
+</include>
+
+1. Icon specifications should be attached only to the first element of a list item (for the example above, the icon specification should be attached to the heading, not the block of text below the heading).
+2. A blank line must separate a heading from the content that follows it.
```

Comment:
Is `2.` required? Was able to get it working without:

![image](https://github.com/MarkBind/markbind/assets/3306138/9fd7f295-6823-4f9a-8ce9-6c8e2d25be0b)

Reply:
might have deleted the wrong one

Reply:
Oops fixed.
2. is not required.